crates/sui/tests/cli_tests.rs

```diff
@@ -134,7 +134,6 @@ async fn test_addresses_command() -> Result<(), anyhow::Error> {
     let test_cluster = TestClusterBuilder::new().build().await;
     let mut context = test_cluster.wallet;
 
-    // Add 3 accounts
```

Comment:
// Add 4 accounts

Reply:
Hmm, I think this line got removed by accident, not sure how. It is 3 accounts (loop inclusive of 0, exclusive of 3), so I will just revert the change.